apps/fabric-website/src/SiteDefinition/SiteDefinition.tsx

```diff
@@ -34,6 +21,26 @@ export const SiteDefinition: ISiteDefinition<Platforms> = {
       component: () => <LoadingComponent title="Welcome to Microsoft UI Fabric" />,
       getComponent: cb => require.ensure([], require => cb(require<any>('../pages/HomePage/HomePage').HomePage))
     },
+    {
+      title: 'Get started',
+      url: '#/get-started',
```

Comment:
Can you make sure it works if someone navigates directly to `#/get-started` (not `#/get-started/web`)?